docs/runtime/on-device-compilation.md

```diff
@@ -0,0 +1,202 @@
+# On-Device Compilation
+
+ONERT supports on-device compilation - on-device quantization and on-device code generation.
+
+## On-device quantization
+
+ONERT supports on-device quantization for the **float32** model. On-device quantization has two mode - full quantization and weight-only quantization.
+
+### Weight-only quantization
+
+Weight-only quantization quantizes only weights of the model. The activation is still in float32 precision. This mode is useful when the model size reduction is more important than the inference speedup.
+
+For weight-only quantization, follow below steps:
+- Load float32 model
+- Set quantization type for weight-only quantization
+- Set path to save quantized model
+- Call quantize API to perform quantization
+
+```cpp
+// Load float32 model
+nnfw_load_model_from_file(session, pkg_path);
+
+// Set quantization type: weight-only, symmetric, int8
+nnfw_set_quantization_type(session, NNFW_QUANTIZE_TYPE_WO_I8_SYM);
+
+// Set path to save quantized model
+nnfw_set_quantized_model_path(session, quantized_model_path);
+
+// Quantize model
+nnfw_quantize(session);
+
+// Run model for inference with quantized model
+nnfw_run(session);
+```
+
+When the model is quantized, you can use the quantized model because the quantized model is loaded automatically after quantization. You don't need to load the quantized model explicitly.
+
+
+### Full quantization
+
+Full quantization quantizes both weights and activations of the model. This mode is useful when the inference speedup is more important and specific runtime backend requires quantized model. To quantize activation, runtime should gather information about activation range during the execution of the model. Therefore, it needs to run the model enough times to get accurate activation range.
```

Comment:
@hseok-oh Is inference latency slower in quantized model ?

Reply:
I removes comment about performance.